gemac/management.py

```diff
@@ -37,12 +36,12 @@ def management(hostintf, mdiointf, reset):
     Args:
         hostintf - Instance of 'HostManagementInterface' class in interfaces.
         mdiointf - Instance of 'MDIOInterface' class in interfaces.
+        configregs - List of 10 32-bits wide Configuration Registers.
+        addrtable - List of 4 48-bits wide MAC Addresses to be used by
+            Address Filter.
         reset: Asynchronous reset Signal from Host
```

Comment:
Just a comment change, yes just indicate that it is a system reset.  The reset needs some thought here, because you have different clocks you typically want a reset per clock domain, a reset that is set async but released sync to the clock.